guides/hack/11-built-in-types/36-enum-class-label.md

```diff
@@ -0,0 +1,102 @@
+# Disclaimer
+This feature is on its way to be deployed. Most recent HHVM versions no longer need the `EnableUnstableFeatures` file attribute
+to support this. More information available at  https://github.com/hhvm/user-documentation/pull/1122
+
+## Values v. Bindings
+
+With [enum types](enumerated-types) and [enum classes](enum-class), most of the focus is given to their values.
```

Comment:
Can you use the full path for links?
* (enumerated-types) -> (/hack/built-in-types/enum)
* (enum-class) -> (/hack/built-in-types/enum-class)

This format lets our test suite push error notifications if page titles change and someone forgets to update links in other docs pages (like this one).